src/shared/ui/Input/ui/index.tsx

```diff
@@ -23,15 +32,39 @@ const Input = (props: IInputCommonProps) => {
     onFormChange(value)
   }
 
+  const onIconClick = () => {
```

Comment:
This should be handled outside of this component

Reply:
Done